### PR TITLE
dev-python/notebook: Version bump to 5.2.1

### PR DESCRIPTION
**Pull request by marbre**
Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

File: dev-python/notebook/notebook-5.2.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		dev-python/coverage[${PYTHON_USEDEP}]
+	)
+	doc? (
+		app-text/pandoc
+		>=dev-python/ipython-4.0.0[${PYTHON_USEDEP}]
+		>=dev-python/sphinx-1.3.6[${PYTHON_USEDEP}]
+		dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
+		=dev-python/nbsphinx-0.2.14[${PYTHON_USEDEP}]
+		test? (
+			$(python_gen_cond_dep 'dev-python/nbval[${PYTHON_USEDEP}]' 'python3*')
```
Comment (marbre) [inline dev-python/notebook/notebook-5.2.1.ebuild]: nbval is only available for python3

File: dev-python/notebook/notebook-5.2.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+python_test() {
+	nosetests \
+		--verbosity=3 \
+		notebook || die
+
+	if use doc && python_is_python3; then
+		py.test --nbval --current-env docs || die
+	fi
```
Comment (marbre) [inline dev-python/notebook/notebook-5.2.1.ebuild]: Is this acceptable?

**Comment by monsieurp** [comment]
@SoapGentoo ping

**Comment by flappyports** [comment]
@marbre Do you still intend to work on this PR?

**Comment by marbre** [comment]
@flacjacket It is actually finished as long as no changes are requested.

**Comment by flacjacket** [comment]
This PR does not apply cleanly, it will need to be rebased/merged with master.

Why not bump up to 5.4.1?

**Comment by marbre** [comment]
@flacjacket Doesn't makes sense to me to rebase (as the only conflict is with the Manifest), as long as I don't know if any further changes are requested. I was actually uncertain regarding some of the changes I introduced (see my original comments).

This PR originally was and still is a bump. Of course there were new releases in the meantime and normally I send a new pull request after a new release. Hopefully, you don't get this wrong, but to me this is more an issue of not timely merged PRs.

**Comment by SoapGentoo** [comment]
@marbre please rebase, and also use `--ignore-date` to freshen up the dates

**Comment by marbre** [comment]
@SoapGentoo Done.

**Pull request closed without merging**